lib/APIPlugin.js

```diff
@@ -40,7 +41,7 @@ const REPLACEMENT_TYPES = {
 class APIPlugin {
 	/**
 	 * Apply the plugin
-	 * @param {TODO} compiler Webpack Compiler
+	 * @param {Compiler} compiler Webpack Compiler
 	 * @returns {void}
```

Comment:
If I remove the `@returns {void}` as defined in the typings guide, I get:

![image](https://user-images.githubusercontent.com/966368/41672141-85ac4e08-74b9-11e8-8169-bc664a95937f.png)